tests/test_manager.py

```diff
@@ -10,6 +10,8 @@
 # License version 3 and under the 3-clause BSD license. For more
 # information, see LICENSE.AGPL and LICENSE.BSD.
 """Unit tests for the :mod:`flask_restless.manager` module."""
+from json import loads
```

Comment:
Ahaa! This is what caused the py3 breakages! 

Sorry, I still haven't had much time to venture into python 3 yet :(